### ISSUE TITLE
Tokens created via CreateToken are not valid for use

### ISSUE THREAD

**Issue by lukef** [GetStream/stream-chat-go#69]
Tokens created using `CreateToken` are showing as malformed or as having an invalid Secret. The error is present when using both the CLI and the client SDKs.

Error message example (from CLI):
```
Error: Malformed JWT token or Stream API secret.
```

**Comment by ferhatelmas** [comment]
@lukef Could you show a sample of what you did? i.e. client init and create token call. Thanks!

**Comment by lukef** [comment]
Sure thing. 

The client is being created with:
```
client, clientErr := stream_chat.NewClient(streamAppID, []byte(streamAppSecret))
```
Then to create a token, we do:
```
token, tokenErr := us.streamClient.CreateToken(
	user.ID, time.Now().UTC().AddDate(100, 0, 0),
)
```
We originally set expiry to 100 years because we couldn't pass a nil value but we tried it with a zero value also and have the same issue. With a zero value it looks like:

```
token, tokenErr := us.streamClient.CreateToken(
	user.ID, time.Time{},
)
```

I also noticed that if i check the JWT the expiry can have values like:
```
"exp": 4742743139.000001
```

Not sure if that is related or not.

**Comment by ferhatelmas** [comment]
@lukef your issue should be fixed in `v2.2.1`. Please let me know otherwise. Thanks for the report.
